resources/assets/js/helpers/router.ts

```diff
@@ -37,17 +37,21 @@ export const useUrlHash = (): void => {
 };
 
 export const useRouter = (
-  routes: Routes<any, React.ReactElement>,
+  routes: Routes<any, any>,
 ): React.ReactElement | null => {
   const location = useLocation();
   const router = useMemo(() => new UniversalRouter(routes), [routes]);
   const [component, setComponent] = useState<React.ReactElement | null>(null);
 
   // Render the result of routing
   useEffect((): void => {
-    router
-      .resolve(location.pathname)
-      .then((result): void => setComponent(result));
+    router.resolve(location.pathname).then((result): void => {
+      const { defaultMessage } = result.title.props;
+      const h1 = document.getElementById("h1");
```

Comment:
It seems odd to me to use "h1" as an id. Why not just search for the first <h1> element?

Reply:
```js
document.querySelector('h1');
```

Reply:
Lol. For some reason I thought you only had access to innerHTML with getElementById.